### ISSUE TITLE
color shift when saving with GLMakie.save

### ISSUE THREAD

**Issue by koehlerson** [MakieOrg/Makie.jl#2865]
If I record a gif, the background color is correctly captured, see e.g. this example where the background color of my slides match the gif background color:

![grafik](https://user-images.githubusercontent.com/27497290/232776360-72f9f86b-47b5-453b-a951-720fbfc531d0.png)

However, using `GLMakie.save` renders a slightly different color than `record`:
![grafik](https://user-images.githubusercontent.com/27497290/232776547-f6360837-5952-4af4-b433-a1250351691c.png)

**Comment by jkrumbiegel** [comment]
can you read the image back in and check if the rgb value of the background is actually incorrect? because images can be displayed differently depending on color space etc., which you'd want to exclude as a factor

**Comment by asinghvi17** [comment]
Huh.  My color meter is showing that the top image is #181818 and the bottom is #1A1A1A.  I am leaning towards a browser issue but have no idea why that might be the case!  Could you send me the PNG after gimp export, just drag and drop it in to the comment field of the github issue?  Will inspect it on my machine and try to see what the difference is.

**Comment by koehlerson** [comment]
This is before gimp reexport

https://github.com/koehlerson/MakieCon/blob/58a3844030cc7b1376adc8065706f57c70becd65/assets/arrows-incompressible.png

and this is afterwards

https://github.com/koehlerson/MakieCon/blob/bca0a64463cd419683bef96c5e86bba4e4df3a15/assets/arrows-incompressible.png

**Comment by koehlerson** [comment]
But again, if you import these two, they probably both will show you the same value in gimp (at least that's for me the case and I don't see a alpha channel with the color picker tool). The issue only appears if I include them in e.g. slides or in general displaying it with the browser I guess

**Comment by asinghvi17** [comment]
Huh!  Can confirm that they show the same value...can you try saving them with CairoMakie and see if that has this issue?  (just run `save("filename.png", fig; backend = CairoMakie)`

**Comment by koehlerson** [comment]
CairoMakie save works out of the box without the gimp extra steps

**Comment by jkrumbiegel** [comment]
This might be a color space embedding issue. The color space implementation factors in the moment you compare via screenshots. The same colors can be rendered differently depending on the color space that is assumed.  Maybe the gimp edit writes the color space information. Can you upload the original glmakie png and the gimp png? One could duff these with a metadata tool and see where the deciding factor is.

**Comment by koehlerson** [comment]
> This is before gimp reexport
> 
> https://github.com/koehlerson/MakieCon/blob/58a3844030cc7b1376adc8065706f57c70becd65/assets/arrows-incompressible.png
> 
> and this is afterwards
> 
> https://github.com/koehlerson/MakieCon/blob/bca0a64463cd419683bef96c5e86bba4e4df3a15/assets/arrows-incompressible.png

The first is the GLMakie png and the second link the gimp png

**Comment by jkrumbiegel** [comment]
Yup looks like after has some different `gamma` related metadata, that would directly affect lightness information if the gamma curve was different.

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/22495855/233709422-4a1cda56-e8c1-4634-9b88-2daf99e5a5b3.png">

**Comment by jkrumbiegel** [comment]
Looks like the original has some gamma / white point information in it and the edited one not. So I assume both files are interpreted with different gamma and that gives the lightness difference.

<img width="1441" alt="image" src="https://user-images.githubusercontent.com/22495855/233709812-51ac1f5c-1d5e-46c1-916b-e6577b1b1fab.png">
